pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -78,8 +82,20 @@ func newNodeLabeller(clusterConfig *virtconfig.ClusterConfig, clientset kubecli.
 		volumePath:              volumePath,
 		domCapabilitiesFileName: "virsh_domcapabilities.xml",
 		hostCPUModel:            hostCPUModel{requiredFeatures: make(map[string]bool, 0)},
+		nodeInformer:            nodeInformer,
+		kvInformer:              kvInformer,
 	}
 
+	n.nodeInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    func(_ interface{}) { n.requeueHost() },
```

Comment:
Well, TBH I'm not sure if it's needed for the first node-labeller run. In other words, when Kubevirt is installed, what would cause a controller to start running for the first time? Isn't the node on which virt-handler runs on considered as "added" by this informer?

In any case, it might be redundant but I don't think it's harmful

Reply:
I'm not sure about it but if this is the case kubevirtCR will be considered as "added" by this informer and that would solve this problem.

Reply:
This already exists before this PR was introduced.
So I guess that the only thing that needs to be done is to change Kubevirt CR in the tests in order to wake node-labeller.

Thanks @xpivarc, I have seemed to miss that. Wasn't familiar with these callbacks before.

p.s.
Do you think we should keep the tests introduced in this PR? If yes, I will convert it to draft and work on that. If not, I will close this PR.